src/features/Account/ManageAccountContainer.tsx

```diff
@@ -160,29 +159,13 @@ class ManageAccountContainer extends React.Component<
   private renderForm() {
     const { mode, accountDetails } = this.state;
     return (
-      // <HorizontalSpacer
-      //   paddingLeft={mode === ManageAccountModes.CREATE ? '0' : '18%'}
-      // >
-      <MaxWidthBox m={'auto'} maxWidth={'500px'}>
+      <MaxWidthBox paddingLeft={'100px'} pr={'50px'} maxWidth={'500px'}>
```

Comment:
confusing syntax here, I think you're using `pr` from rebass but the other declarations we defined in `MaxWidthBox`.

pending a future refactor of our styled components, can you declare `paddingRight` in `MaxWidthBox` for now?